src/banana/data/theory_template.yaml

```diff
@@ -45,4 +45,4 @@ Comments: "LO baseline for small-x res"
 global_nx: 0
 EScaleVar: 1
 ModSV: null
-alphaem_running: 1
+alphaem_running: 0
```

Comment:
Consider using a boolean.

`0` and `1` are an APFEL legacy, since we preserved full compatibility, translating only on our side.
But this parameter didn't exist before, so you can use a boolean as well.

In any case, at some point we'll refactor all the runcards, so it is not extremely relevant.